diff --git a/bazel/grpc_build_system.bzl b/bazel/grpc_build_system.bzl
index 3cda5a6..b8e94ce 100644
--- a/bazel/grpc_build_system.bzl
+++ b/bazel/grpc_build_system.bzl
@@ -27,8 +27,6 @@
 Contains macros used throughout the repo.
 """
 
-load("@build_bazel_rules_apple//apple:ios.bzl", "ios_unit_test")
-load("@build_bazel_rules_apple//apple/testing/default_runner:ios_test_runner.bzl", "ios_test_runner")
 load("@com_google_protobuf//bazel:upb_proto_library.bzl", "upb_proto_library", "upb_proto_reflection_library")
 load("//bazel:cc_grpc_library.bzl", "cc_grpc_library")
 load("//bazel:copts.bzl", "GRPC_DEFAULT_COPTS")
@@ -265,34 +263,7 @@ def ios_cc_test(
       tags: The tags to apply to the test.
       **kwargs: All other arguments to apply.
     """
-    test_lib_ios = name + "_test_lib_ios"
-    ios_tags = tags + ["manual", "ios_cc_test"]
-    test_runner = "ios_x86_64_sim_runner_" + name
-    ios_test_runner(
-        name = test_runner,
-        device_type = "iPhone X",
-    )
-    if not any([t for t in tags if t.startswith("no_test_ios")]):
-        native.objc_library(
-            name = test_lib_ios,
-            srcs = kwargs.get("srcs"),
-            deps = kwargs.get("deps"),
-            copts = kwargs.get("copts"),
-            data = kwargs.get("data"),
-            tags = ios_tags,
-            alwayslink = 1,
-            testonly = 1,
-        )
-        ios_test_deps = [":" + test_lib_ios]
-        ios_unit_test(
-            name = name + "_on_ios",
-            size = kwargs.get("size"),
-            data = kwargs.get("data"),
-            tags = ios_tags,
-            minimum_os_version = "11.0",
-            runner = test_runner,
-            deps = ios_test_deps,
-        )
+    pass
 
 def expand_poller_config(name, srcs, deps, tags, args, exclude_pollers, uses_polling, uses_event_engine, flaky):
     """Common logic used to parameterize tests for every poller and EventEngine.
